core/utils/contracts/mod.ts

```diff
@@ -1,5 +1,6 @@
-export * from './call.ts';
-export * from './decodeContractExecResult.ts';
-export * from './decodeError.ts';
-export * from './toContractAbiMessage.ts';
-export * from './toRegistryErrorDecoded.ts';
+export * from "./call.ts";
+export * from "./decodeContractExecResult.ts";
+export * from "./decodeError.ts";
+export * from "./toContractAbiMessage.ts";
+export * from "./toRegistryErrorDecoded.ts";
+export * from "./toStatusName.ts";
```

Comment:
maybe we need a `.prettierrc` to avoid inconsistent auto-save?
cc @DoubleOTheven

Reply:
@nicolad You can set Deno as the formatter. It can format on save too.

Reply:
the denoland vscode plugin works well